pixi.toml

```diff
@@ -19,6 +19,9 @@ python = ">=3.11.0,<3.12"
 rattler-build = ">=0.35.5"
 anaconda-client = ">=1.12"
 
+[pypi-dependencies]
+vinca = { git = "https://github.com/RoboStack/vinca.git", rev = "cbb8eba834ce3834df552977d6b08c325a30768e" }
```

Comment:
Can we remove the duplicated `vinca` dep in `feature.beta.pypi-dependencies` to avoid duplication? I would also leave the commented block to easily switch to a local vinca.